packages/office-ui-fabric-react/src/components/Sticky/Sticky.types.ts

```diff
@@ -29,6 +29,12 @@ export interface IStickyProps extends React.Props<Sticky> {
    * @defaultvalue true
    */
   isScrollSynced?: boolean;
+
+  /**
+   * If provided, it will decide the order of stickies
+   * Best Practices: Perf reasons
```

Comment:
Might be worth explaining how to use this new property. Recommend stating that the algorithm evaluates sort order based on the value present for `order` (ascending vs. descending).

And that this is advised as a performance optimization to avoid manual expensive calculations.

Reply:
> Might be worth explaining how to use this new property. Recommend stating that the algorithm evaluates sort order based on the value present for order (ascending vs. descending).
> And that this is advised as a performance optimization to avoid manual expensive calculations.

I will mention this under the best practices section for Sticky.